ui/src/dashboards/components/DashboardsPageContents.tsx

```diff
@@ -9,13 +9,14 @@ import {ErrorHandling} from 'src/shared/decorators/errors'
 import OverlayTechnology from 'src/reusable_ui/components/overlays/OverlayTechnology'
 import {Button, ComponentColor, IconFont} from 'src/reusable_ui'
 
-import {Dashboard, Source} from 'src/types'
+import {Dashboard, Source, RemoteDataState} from 'src/types'
```

Comment:
Thanks for naming this more generically, I think I'll use it for hosts page loading indication as well.